detector.go

```diff
@@ -197,7 +197,7 @@ func mergeTOML(l *log.Logger, out io.Writer, in ...io.Reader) {
 
 type BuildpackOrder []BuildpackGroup
 
-func (bo BuildpackOrder) Detect(c *DetectConfig) (plan []byte, group *BuildpackGroup) {
+func (bo BuildpackOrder) Detect(c *DetectConfig) ([]byte, *BuildpackGroup) {
```

Comment:
Keep named parameters for better documentation.

Reply:
We removed them because they were showing as obfuscated in the debugged

Reply:
I think replacing `plan` and `group` on 202 with `p` and `g` would solve this and keep the godoc more clear, but approving either way.

Reply:
Ah yeah, I agree with @sclevine thinking about the gdocs. Optionally we can just call `return` but that never felt very readable.